src/pandapipes/io/file_io.py

```diff
@@ -150,7 +150,8 @@ def from_json_string(json_string, convert=False, encryption_key=None):
     if encryption_key is not None:
         json_string = decrypt_string(json_string, encryption_key)
 
-    net = json.loads(json_string, cls=PPJSONDecoder, registry_class=FromSerializableRegistryPpipe)
+    net = json.loads(json_string, cls=PPJSONDecoder,
+                     object_hook=partial(pp_hook, registry_class=FromSerializableRegistryPpipe))
```

Comment:
I am wondering why a change is necessary here... is it due to a new version in pandapower? Should we add it to a different PR?

Reply:
I changed it back. No clue, where it came from. Not required.